inc/gui/class-workflow.php

```diff
@@ -126,4 +126,29 @@ public static function get_task_ids_by_workflow( $workflow_id ) {
 		$task_ids = get_post_meta( $workflow_id, 'tasks', true );
 		return $task_ids['tasks'] ?? [];
 	}
+
+	/**
+	 * Helper to get a workflow ID from a slug.
+	 *
+	 * @param string $slug Workflow slug.
+	 * @return int|null
+	 */
+	public static function get_workflow_id_by_slug( string $slug ) : ?int {
```

Comment:
No space before the colon in return type declarations, [per the core standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage).

Reply:
Didn't know that, thanks!